CHANGELOG.md

```diff
@@ -17,11 +18,117 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
-# r2.1
+# r2.2
 
 ## Release Notes
 
 This **public release** contains the definition and documentation of
+* location-verification v2.0.0
+* location-retrieval v0.4.0
+* geofencing-subscriptions v0.4.0
+
+The API definition(s) are based on
+* Commonalities r2.3
+* Identity and Consent Management r2.3
+
+**Changelog since r2.1**
+* Full Changelog with the list of PRs and contributors: https://github.com/camaraproject/DeviceLocation/compare/r2.1...r2.2
+
+## location-verification v2.0.0
+
+**There are breaking changes compared to v1.0.0**: the API use has been simplified for API consumers using a three-legged access token to invoke the API. 
+In these cases the optional `device` parameter MUST NOT be provided, as the subject will be uniquely identified from the access token. 
+In this context also some error response codes have been renamed or replaced to comply with Commonalities 0.5.
+
+### Breaking Changes

```

Comment:
I would not add this section explicitly because some of the other changes below may also be non-backward compatible. With the note above I think it is enough to remark that there are breaking changes.
To be removed from r2.1 as well